businessRules/donations.js

```diff
@@ -1,15 +1,17 @@
 const { v4 } = require('uuid');
 const { createDonation } = require('../models/donations');
-const { getAllIdeas } = require('../models/ideas');
-const { getIdeas } = require('../utils/storage');
+const {
+    ideaThatMatchesIdeaId,
+} = require('../validation/ideaFunctionValidation');
+const {
+    checkIfSumIsANumber,
```

Comment:
This function is too specific. Can you import a more generic function, for example: `validateSum` and inside of `validateSum` you can check if the sum is a number, etc.

Remember that different modules have different responsibilities. And by default, they should hide as much details about these responsibilities as possible. The validation module should not communicate to other modules what exactly it's validating, unless other modules need to know it by design.

Here your business rules module knows "too much" about what validation module does. Instead, business rules module should simply know that it should validate the sum, but it should not know what exactly it's validating and why - it's not relevant to it's responsibilities.

Reply:
Mmm, I think I've understood you. Fixed.